proto/v1/vulnerability.proto

```diff
@@ -207,6 +207,16 @@ message VulnerabilityOccurrence {
     // when that is available and note provider assigned severity when it is not
     // available.
     grafeas.v1.Severity effective_severity = 9 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+    // Indicates the lcation at which a package was found.
+    message FileLocation {
```

Comment:
Would it make sense to define the message FileLocation outside of VulnerabilityOccurrence so it could be used in other protos, for example SPDX PackageInfoOccurrence?

Reply:
I have moved it to common.proto, so people will not need to include vulnerability.proto if they want to use FileLocation.